demos/checkmarx/README.md

```diff
@@ -0,0 +1,37 @@
+# configure checkmarx-plugin
+
+## sample configuration
+Here is a sample configuration using a credential. It is also possible to specify a username and password in the cxscanbuilder section. 
+The filterPattern is the default pattern provided by checkmarx documentation.
+
+```yaml
+credentials:
+  system:
+    domainCredentials:
+      - credentials:
+        - usernamePassword:
+          scope: GLOBAL
+          id: checkmarx_credential
+          description: "login/pwd for checkmarx"
+          username: jenkins_checkmarx
+          password: "StrongPassword"
+
+unclassified:
+  cxscanbuilder:
+    credentialsId: checkmarx_credential
+    serverUrl: https://checkmarx.url.tld
+    enableCertificateValidation: true
+    filterPattern: '!**/_cvs/**/*, !**/.svn/**/*,   !**/.hg/**/*,   !**/.git/**/*,  !**/.bzr/**/*, !**/bin/**/*,
```

Comment:
please use literal
```suggestion
    filterPattern: -|
    my multiline string
```

Reply:
Thanks @casz for pointing the email aliases issue. I have updated both configurations (local and githib ;) )

I totally forgot about the litteral syntax ! It should be cleaner.
I will push it on monday, after some validation with a test jenkins.

Reply:
@kleph if you follow @v1v our CI will do the validation for you